mm2src/mm2_main/src/lp_wallet.rs

```diff
@@ -550,3 +532,64 @@ pub async fn get_wallet_names_rpc(ctx: MmArc, _req: Json) -> MmResult<GetWalletN
         activated_wallet: activated_wallet.clone(),
     })
 }
+
+/// `SeedPasswordUpdateRequest` represents a request to update
+/// the password for the seed storage.
+/// It includes the current password and the new password to be set.
+#[derive(Debug, Deserialize)]
+pub struct SeedPasswordUpdateRequest {
+    /// The current password for the seed storage.
+    pub current_password: String,
+    /// The new password to replace the current password.
+    pub new_password: String,
+}
+
+/// `SeedPasswordUpdateResponse` represents the result of a
+/// password update request.
+/// It contains a boolean indicating whether the operation was successful.
+#[derive(Serialize)]
+pub struct SeedPasswordUpdateResponse {
+    /// `true` if the password update was successful, `false` otherwise.
+    successful: bool,
+}
+
+/// RPC function to handle a request for updating the seed storage password.
+///
+/// # Arguments
+/// - `ctx`: The shared context (`MmArc`) for the application.
+/// - `req`: The `SeedPasswordUpdateRequest` containing the current and new passwords.
+///
+/// # Example
+/// ```ignore
+/// let request = SeedPasswordUpdateRequest {
+///     current_password: "old_password".to_string(),
+///     new_password: "new_password".to_string(),
+/// };
+/// let response = update_seed_storage_password_rpc(ctx, request).await?;
+/// assert!(response.successful);
+/// ```
```

Comment:
We agreed not to describe function parameters in the function documentation. 
Instead, provide documentation for the parameter types themselves (e.g. above SeedPasswordUpdateRequest). 
It actually makes docs more readable. If I want to know nuances about each param or result I will go to its type doc comment.

Reply:
removed